src/core_plugins/kibana/public/home/components/home.js

```diff
@@ -56,7 +57,7 @@ export function Home({ addBasePath, directories }) {
     return (
       <div className="kuiVerticalRhythm">
         <KuiCardGroup>
-          <KuiCard style={cardStyle}>
+          <KuiCard style={cardStyle} className="euiPanel">
```

Comment:
This class works for now. I'll replace with a EuiCard component once I have one built.